client/components/domains/domain-search-results/index.jsx

```diff
@@ -83,30 +83,7 @@ class DomainSearchResults extends React.Component {
 					onButtonClick={ this.props.onClickResult }
 				/>
 			);
-		} else if (
```

Comment:
Won't do, because we rely on the statuses, otherwise we'll show a message for everything, like `andrija.m`.